infra/docker/Dockerfile

```
@@ -1,42 +1,45 @@
+# Copy Air configuration
+COPY infra/config/.air.toml ./infra/config/.air.toml
 
 # Create non-root user for security
-RUN addgroup -g 1001 -S appgroup && \
-    adduser -u 1001 -S appuser -G appgroup
+RUN adduser -D -s /bin/sh -u 1001 appuser
 
-COPY --from=builder /app/main .
-COPY .env.example .env
+# Change ownership of application files to non-root user
+RUN chown -R appuser:appuser /app
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)